app/models/user.rb

```diff
@@ -52,10 +53,10 @@ def authorized?
   end
 
   def synchronized_batches
-    batches.select(&:synchronized?)
+    (batches.select(&:synchronized?) + marc_batches.select(&:synchronized?)).sort_by(&:created_at)
   end
 
   def unsynchronized_batches
-    batches.reject(&:synchronized?)
+    (batches.reject(&:synchronized?) + marc_batches.reject(&:synchronized?)).sort_by(&:created_at)
```

Comment:
what is the difference between synchronized and unsynchronized batches?

Reply:
Unsynchronized batches aren't yet persisted to Alma (or ASpace)